src/crate/crash/command.py

```diff
@@ -368,6 +378,22 @@ def _verify_connection(self, verbose=False):
                 # check for failing node and cluster checks
                 built_in_commands['check'](self, startup=True)
 
+    def _fetch_session_info(self):
+        if self.is_conn_available() \
+                and self.connection.lowest_server_version >= StrictVersion("2.0"):
+            self.cursor.execute("""
+                SELECT settings['license']['enterprise'] FROM sys.cluster;
+            """)
+            user_function = "current_user" if self.cursor.fetchone()[0] else "NULL"
```

Comment:
maybe add a short command that `current_user` is only available if enterprise is enabled.